lib/rules/import-notation/index.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const valueParser = require('postcss-value-parser');
+
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+const setAtRuleParams = require('../../utils/setAtRuleParams');
+const getAtRuleParams = require('../../utils/getAtRuleParams');
+const atRuleParamIndex = require('../../utils/atRuleParamIndex');
+
+const ruleName = 'import-notation';
+
+const messages = ruleMessages(ruleName, {
+	expected: (importUrlOrString, primary) => `Expected "${importUrlOrString}" to be a "${primary}"`,
```

Comment:
```suggestion
	expected: (unfixed, fixed) => `Expected "${unfixed}" to be "${fixed}"`,
```

My bad, I got this wrong in [the blueprint](https://github.com/stylelint/stylelint/issues/4055#issuecomment-1014768867).

We use the:
- "Expected x to be y" message format when showing the fixed values, e.g. https://github.com/stylelint/stylelint/blob/9d045d928699c0d84b86e08a3d8b035343b06ca4/lib/rules/alpha-value-notation/index.js#L18
- "Expected primary option" message format when we don't, e.g. https://github.com/stylelint/stylelint/blob/9d045d928699c0d84b86e08a3d8b035343b06ca4/lib/rules/color-function-notation/index.js#L17

We should use the former here as:
- it's generally more useful for users
- the URL/string parts of `@import` rules are generally short

e.g. "Expected '"foo.css"' to be 'url("foo.css")' (import-notation)"

We'll need to use single quotes if there are double quotes in the value, and vice-versa.